tests/e2e/test_api.py

```diff
@@ -81,11 +183,19 @@ def test_query_call_without_payload():
     # so let's do just primitive check
     assert "missing" in response.text
 
+    counter = get_rest_api_counter_value(client, "rest_api_calls_total", endpoint, 422)
```

Comment:
At first sight, I thought the `422` is the actual counter value, not the status code. I would do `status_code=422`, but up to you. Not a big deal.

Reply:
yup that needs change, indeed

Reply:
fixed, now it should be more readable